assets/react/components/CommunityCalendar.jsx

```diff
@@ -1,17 +1,84 @@
 import React from 'react';
+import moment from 'moment';
+import { API_BASE_URL } from '../constants'
 
 
-const CommunityEvent = ({event}) => (
-  <div class="card">
-    { event.image && <img src={event.image} class="card-img-top" alt="image of event" /> }
-    <div class='card-body'>
-      <h5 class='card-title'>{event.title}</h5>
-      <p class='card-text'>{event.local_datetime}</p>
-      <p class='card-text'>{event.description}</p>
-      <a href={event.link} class="btn btn-primary" target="_blank" rel="nofollow" >More info</a>
+const FeaturedCommunityEvent = ({event}) => {
+  const eventDate = moment(event.local_datetime);
+
+  return(
+    <div className="featured-event bg-dark mt-2 mb-5 pb-5">
+      <div className="container">
+        <div className="d-flex justify-content-end">
+          <div className="featured-tag p-2">
+            <span>Next event:</span><span className="bold">{eventDate.format('MMM DD, YYYY')}</span>
```

Comment:
@s-kennedy - I think we need to move away from doing explicit date formatting as we're working increasingly with different locales. People expect to see dates and times in a familiar way, like day, month, year or 24 hour times. Before I was hoping that we could maybe choose a compromise that would be acceptable - like displaying month names to avoid ambiguity, but I don't think that is good enough and native javascript + browsers have the tools available to localize dates and times.

Have a look at what I've done for the facilitator dash [1] [2] and also the Berlin site [3].

Additionally, moment.js as a dependency is a real pain. Unless we're doing date arithmetic, I would rather avoid moment.

Let's chat about it in WIP today or tech call tomorrow

[1]: https://github.com/p2pu/learning-circles/blob/community-calendar/frontend/components/dashboard/EventsTable.jsx#L7
[2]: https://github.com/p2pu/learning-circles/blob/community-calendar/frontend/helpers/i18n.js
[3]: https://github.com/p2pu/zlb/blob/gh-pages/assets/jsx/components/learning-circle-signup.jsx#L21-L24